extensions/attributegen/testdata/server.yaml

```diff
@@ -104,14 +104,14 @@ static_resources:
           - name: istio.attributegen
             typed_config:
               '@type': type.googleapis.com/udpa.type.v1.TypedStruct
-              type_url: type.googleapis.com/envoy.config.filter.http.wasm.v2.Wasm
+              type_url: type.googleapis.com/envoy.extensions.filters.http.wasm.v3.Wasm
               value:
                 config:
                   configuration: |
-                    {
+                    { "debug": "true",
                       "attributes": [
                         {
-                          "output_attribute": "istio.operationId",
+                          "output_attribute": "istio_operationId",
```

Comment:
is there a separate PR for fixing the docs re: namespacing of attributes?

Reply:
Will send another PR.